.evergreen/generate_evergreen_tasks.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs');
 const yaml = require('js-yaml');
+const { mongoshTasks } = require('./generate_mongosh_tasks');
 
 const MONGODB_VERSIONS = ['latest', 'rapid', '6.0', '5.0', '4.4', '4.2', '4.0', '3.6'];
```

Comment:
Since the mongosh tasks were broken out into their own generation script, would it make sense to do the same with the versioning code? This way it could be imported here and in the mongosh script, and then for example `LOWEST_LTS` or `LATEST_LTS` could be used in both places, and only need to ever be updated in 1 file and not 2.

Reply:
Yeah, I can do that.  Were you thinking all the constants?

```javascript
const MONGODB_VERSIONS = ['latest', 'rapid', '6.0', '5.0', '4.4', '4.2', '4.0', '3.6'];
const versions = [
  { codeName: 'fermium', versionNumber: 14 },
  { codeName: 'gallium', versionNumber: 16 },
  { codeName: 'hydrogen', versionNumber: 18 }
];
const NODE_VERSIONS = versions.map(({ codeName }) => codeName)
NODE_VERSIONS.sort();
const LOWEST_LTS = NODE_VERSIONS[0];
const LATEST_LTS = NODE_VERSIONS[NODE_VERSIONS.length - 1];

const TOPOLOGIES = ['server', 'replica_set', 'sharded_cluster'];
const AWS_AUTH_VERSIONS = ['latest', '6.0', '5.0', '4.4'];
const TLS_VERSIONS = ['latest', '6.0', '5.0', '4.4', '4.2'];

const DEFAULT_OS = 'rhel80-large';
```

Reply:
Done.